src/PrestaShopBundle/Install/LanguageList.php

```diff
@@ -119,7 +119,7 @@ public function getLanguageIso()
      */
     public function getLanguage($iso = null)
     {
-        if (!$iso) {
+        if (!isset($this->languages[$iso])) {
```

Comment:
@PierreRambaud I tested your modif and validated that the bug doesn't happen any more

However after testing the PR in details and searching for what happens in the code, I don't think the source of the problem is here Indeed we try to access an undefined index in the array but the problem is somewhere else.

It originates from https://github.com/PrestaShop/PrestaShop/blob/8217e88040294ca93a95c0af5825cad5f9e679af/src/PrestaShopBundle/Install/XmlLoader.php#L385

we try to get the countries for each languages installes In this case the lanagues present are `en-US` and `en-GB` The first one works fine because its iso value is `en` however the second one is `gb` which is not avialable in `LanguageList::$languages` which is built based on these files:

https://github.com/PrestaShop/PrestaShop/tree/develop/install-dev/langs

So a more robust fix would be that `XMLLoader::populateEntityCountry` checks that the iso is managed by LanguageList and if not:
- either ignore it
- or try to find the "source" language based on the iso (gb => en) and call the `getCountriesByLanguage` again with the fallback iso code